digitalfabrik/entitlementcard
# 1399: Statistics created cards

### Short description

As a service user you can check card statistic for region (Region Manager) or for project (Project Manager)

### Proposed changes



- add query to get card statistics
- add functions to get cards statistics per region or project
- add bar chart for visualization
- add filter bar to customize time period

### Side effects



- none

### Resolved issues



Fixes: #1399 

Unfortunately i couldn't get a unit test working with the bar chart and didn't find any good hints

### Testing
Region Admin
1) Login as Region Admin for bavaria
2) Go to statistics 
3) Only card statistics for this region should be shown
4) Adjust some filters and check error handling of input fields
5) Do the same for Project Admin (you may create some cards for several region)

I was thinking about filtering "0" regions but in reality with real data there won't be (m)any
